CHANGELOG.md

```diff
@@ -1,3 +1,68 @@
+# v1.14.0 (2023-05-10)
+
+## OS Changes
+
+* Update kernel-5.10 to 5.10.178 and kernel-5.15 to 5.15.108 ([#3077])
+* Update admin and control containers ([#3090])
+* Update third party packages and dependencies ([#2991], [#3082])
+* Enable `SCSI_VIRTIO` driver for better hypervisor support ([#3047])
+* Disable panic on hung task for kernel 5.15 ([#3091])
+* Create symlink to `inventory` path using Storewolf ([#3035])
+
+## Orchestrator Changes
+
+### ECS
+
+* Add support for ECS Exec ([#3075])
+
+### Kubernetes
+
+* Add Kubernetes 1.27 variants ([#3046])
+* New Kubernetes settings for: ([#2930], [#2986])
+  * Soft eviction policy
```

Comment:
nit: I don't feel strongly about this, but maybe ...? 
```suggestion
* Add more Kubernetes settings ([#2930], [#2986])
  * Soft eviction policy
```

Reply:
Done!